### ISSUE TITLE
Remote Notification still comes with sound on Android even when device is set to vibration mode

### ISSUE THREAD

**Issue by Gabsys** [zo0r/react-native-push-notification#1407]
# Bug

I have set up remote notification for Android device, however, there's one issue which is the notification still comes with sound even though I already set my device to vibration.

## Environment info

`react-native info` output:

```bash
System:
    OS: macOS Mojave 10.14.6
    CPU: (4) x64 Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz
    Memory: 39.16 MB / 8.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 13.7.0 - /usr/local/bin/node
    Yarn: 1.17.0 - /usr/local/bin/yarn
    npm: 6.14.2 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      Android NDK: 20.0.5594570
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.6010548
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    @react-native-community/cli: ^2.9.0 => 2.10.0 
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5 
  npmGlobalPackages:
    react-native-cli: 2.0.1
```

Library version: 3.2.1

## Steps To Reproduce

1. Refer to `Usage` section of the library. I set up exactly like the Usage section.

Describe what you expected to happen:

1. When device is set to vibration/silent mode, the notification should be silent as well

**Comment by Dallas62** [comment]
Hi,
Can you test with 3.3.1?

**Comment by Gabsys** [comment]
Seems like it is solved in 3.3.1. Thanks

**Comment by vaibhavhrt** [comment]
working as expected if I don't provide `soundName`. But when using custom sound, it plays always, even if phone is in vibration or silent mode.
